examples/07-EMIT/ComputeChannelSeparation.py

```diff
@@ -0,0 +1,443 @@
+"""
+EMIT: Compute required channel separation
+-------------------------------------------
+This example shows how you can use PyAEDT to open an AEDT project with
+an EMIT design and analyze the results to determine the required channel
+separation for overlapping bands. 
+"""
+###############################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports.
+import os
+import sys
+import time
+import subprocess
+import pyaedt
+from pyaedt import Emit
+from pyaedt.emit import Revision
+from pyaedt.emit import Result
+from pyaedt.modeler.circuits.PrimitivesEmit import EmitComponent
+
+# Check to see which Python libraries have been installed
+reqs = subprocess.check_output([sys.executable, '-m', 'pip', 'freeze'])
+installed_packages = [r.decode().split('==')[0] for r in reqs.split()]
+
+# Install required packages if they are not installed
+def install(package): 
+    subprocess.check_call([sys.executable, '-m', 'pip', 'install', package])
+
+# Install any missing libraries
+required_packages = ['plotly', 'tqdm', 'matplotlib', 'numpy']
+for package in required_packages:
+    if package not in installed_packages:
+        install(package)
+
+# Import required modules
+import plotly.graph_objects as go
+from tqdm.notebook import tqdm
+    
+from matplotlib import pyplot as plt
+plt.ion() # enables interactive mode so plots show immediately
```

Comment:
```suggestion
plt.ion() # Enables interactive mode so plots show immediately
```
Only making this edit so that the comment starts with an initial capital letter as is done throughout this example.